src/app/routes/utils/fetchPageData/index.js

```diff
@@ -32,18 +37,11 @@ const handleResponse = url => async response => {
 
   if (upstreamStatusCodesToPropagate.includes(status)) {
     if (status === STATUS_NOT_FOUND) {
-      logger.error(
-        JSON.stringify(
-          {
-            event: 'data_response_404',
-            status,
-            message: `Data not found when requesting ${url}`,
-            url,
-          },
-          null,
-          2,
-        ),
-      );
+      logger.error(DATA_NOT_FOUND, {
+        message: `Data not found when requesting ${url}`,
```

Comment:
```suggestion
```
We can get rid of this message since the event name and url being present is quite self-explanatory.

Reply:
Good shout - done!